[2025-06-16T14:37:42.620+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-16T14:37:42.722+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_brazil_pipeline.run_etl_script manual__2025-06-16T14:37:26.407631+00:00 [queued]>
[2025-06-16T14:37:42.742+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_brazil_pipeline.run_etl_script manual__2025-06-16T14:37:26.407631+00:00 [queued]>
[2025-06-16T14:37:42.744+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-06-16T14:37:42.787+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): run_etl_script> on 2025-06-16 14:37:26.407631+00:00
[2025-06-16T14:37:42.825+0000] {warnings.py:110} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=347) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-16T14:37:42.826+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'etl_brazil_pipeline', 'run_etl_script', 'manual__2025-06-16T14:37:26.407631+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/etl_brazil_pipeline.py', '--cfg-path', '/tmp/tmprng8qxn_']
[2025-06-16T14:37:42.839+0000] {standard_task_runner.py:63} INFO - Started process 361 to run task
[2025-06-16T14:37:42.840+0000] {standard_task_runner.py:91} INFO - Job 67: Subtask run_etl_script
[2025-06-16T14:37:43.476+0000] {warnings.py:110} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-06-16T14:37:44.075+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_brazil_pipeline.run_etl_script manual__2025-06-16T14:37:26.407631+00:00 [running]> on host 6914c8f0cd20
[2025-06-16T14:37:45.194+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='vaishnavi' AIRFLOW_CTX_DAG_ID='etl_brazil_pipeline' AIRFLOW_CTX_TASK_ID='run_etl_script' AIRFLOW_CTX_EXECUTION_DATE='2025-06-16T14:37:26.407631+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-16T14:37:26.407631+00:00'
[2025-06-16T14:37:45.216+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-16T14:37:45.390+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-06-16T14:37:45.410+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/airflow/src/etl/brazil_etl.py']
[2025-06-16T14:37:45.558+0000] {subprocess.py:86} INFO - Output:
[2025-06-16T14:37:58.861+0000] {subprocess.py:93} INFO - /home/airflow/.local/lib/python3.12/site-packages/pandera/_pandas_deprecated.py:146: FutureWarning: Importing pandas-specific classes and functions from the
[2025-06-16T14:37:58.869+0000] {subprocess.py:93} INFO - top-level pandera module will be **removed in a future version of pandera**.
[2025-06-16T14:37:58.871+0000] {subprocess.py:93} INFO - If you're using pandera to validate pandas objects, we highly recommend updating
[2025-06-16T14:37:58.872+0000] {subprocess.py:93} INFO - your import:
[2025-06-16T14:37:58.875+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:37:58.877+0000] {subprocess.py:93} INFO - ```
[2025-06-16T14:37:58.878+0000] {subprocess.py:93} INFO - # old import
[2025-06-16T14:37:58.880+0000] {subprocess.py:93} INFO - import pandera as pa
[2025-06-16T14:37:58.881+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:37:58.883+0000] {subprocess.py:93} INFO - # new import
[2025-06-16T14:37:58.885+0000] {subprocess.py:93} INFO - import pandera.pandas as pa
[2025-06-16T14:37:58.886+0000] {subprocess.py:93} INFO - ```
[2025-06-16T14:37:58.888+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:37:58.889+0000] {subprocess.py:93} INFO - If you're using pandera to validate objects from other compatible libraries
[2025-06-16T14:37:58.890+0000] {subprocess.py:93} INFO - like pyspark or polars, see the supported libraries section of the documentation
[2025-06-16T14:37:58.892+0000] {subprocess.py:93} INFO - for more information on how to import pandera:
[2025-06-16T14:37:58.893+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:37:58.895+0000] {subprocess.py:93} INFO - https://pandera.readthedocs.io/en/stable/supported_libraries.html
[2025-06-16T14:37:58.896+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:37:58.897+0000] {subprocess.py:93} INFO - To disable this warning, set the environment variable:
[2025-06-16T14:37:58.899+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:37:58.900+0000] {subprocess.py:93} INFO - ```
[2025-06-16T14:37:58.902+0000] {subprocess.py:93} INFO - export DISABLE_PANDERA_IMPORT_WARNING=True
[2025-06-16T14:37:58.903+0000] {subprocess.py:93} INFO - ```
[2025-06-16T14:37:58.905+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:37:58.906+0000] {subprocess.py:93} INFO -   warnings.warn(_future_warning, FutureWarning)
[2025-06-16T14:38:17.965+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:17.967+0000] {subprocess.py:93} INFO - üì• Reading /opt/airflow/data/raw/olist_customers_dataset.csv
[2025-06-16T14:38:17.969+0000] {subprocess.py:93} INFO - üßπ Cleaning 'customers' dataset...
[2025-06-16T14:38:17.971+0000] {subprocess.py:93} INFO - üóÑÔ∏è  Loading 'customers' into PostgreSQL...
[2025-06-16T14:38:17.972+0000] {subprocess.py:93} INFO - ‚ùå ERROR while processing /opt/airflow/data/raw/olist_customers_dataset.csv ‚Üí (psycopg2.errors.DependentObjectsStillExist) cannot drop table customers because other objects depend on it
[2025-06-16T14:38:17.974+0000] {subprocess.py:93} INFO - DETAIL:  view stg_customers depends on table customers
[2025-06-16T14:38:17.975+0000] {subprocess.py:93} INFO - HINT:  Use DROP ... CASCADE to drop the dependent objects too.
[2025-06-16T14:38:17.977+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:17.978+0000] {subprocess.py:93} INFO - [SQL:
[2025-06-16T14:38:17.980+0000] {subprocess.py:93} INFO - DROP TABLE customers]
[2025-06-16T14:38:17.982+0000] {subprocess.py:93} INFO - (Background on this error at: https://sqlalche.me/e/14/2j85)
[2025-06-16T14:38:17.983+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:17.985+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:17.987+0000] {subprocess.py:93} INFO - üì• Reading /opt/airflow/data/raw/olist_geolocation_dataset.csv
[2025-06-16T14:38:17.988+0000] {subprocess.py:93} INFO - üßπ Cleaning 'geolocation' dataset...
[2025-06-16T14:38:17.990+0000] {subprocess.py:93} INFO - üóÑÔ∏è  Loading 'geolocation' into PostgreSQL...
[2025-06-16T14:38:17.992+0000] {subprocess.py:93} INFO - ‚ùå ERROR while processing /opt/airflow/data/raw/olist_geolocation_dataset.csv ‚Üí (psycopg2.errors.DependentObjectsStillExist) cannot drop table geolocation because other objects depend on it
[2025-06-16T14:38:17.993+0000] {subprocess.py:93} INFO - DETAIL:  view stg_geolocation depends on table geolocation
[2025-06-16T14:38:17.995+0000] {subprocess.py:93} INFO - HINT:  Use DROP ... CASCADE to drop the dependent objects too.
[2025-06-16T14:38:17.996+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:17.997+0000] {subprocess.py:93} INFO - [SQL:
[2025-06-16T14:38:17.999+0000] {subprocess.py:93} INFO - DROP TABLE geolocation]
[2025-06-16T14:38:18.001+0000] {subprocess.py:93} INFO - (Background on this error at: https://sqlalche.me/e/14/2j85)
[2025-06-16T14:38:18.003+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.005+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.006+0000] {subprocess.py:93} INFO - üì• Reading /opt/airflow/data/raw/olist_orders_dataset.csv
[2025-06-16T14:38:18.007+0000] {subprocess.py:93} INFO - üßπ Cleaning 'orders' dataset...
[2025-06-16T14:38:18.009+0000] {subprocess.py:93} INFO - üóÑÔ∏è  Loading 'orders' into PostgreSQL...
[2025-06-16T14:38:18.010+0000] {subprocess.py:93} INFO - ‚ùå ERROR while processing /opt/airflow/data/raw/olist_orders_dataset.csv ‚Üí (psycopg2.errors.DependentObjectsStillExist) cannot drop table orders because other objects depend on it
[2025-06-16T14:38:18.012+0000] {subprocess.py:93} INFO - DETAIL:  view stg_orders depends on table orders
[2025-06-16T14:38:18.014+0000] {subprocess.py:93} INFO - HINT:  Use DROP ... CASCADE to drop the dependent objects too.
[2025-06-16T14:38:18.016+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.017+0000] {subprocess.py:93} INFO - [SQL:
[2025-06-16T14:38:18.019+0000] {subprocess.py:93} INFO - DROP TABLE orders]
[2025-06-16T14:38:18.021+0000] {subprocess.py:93} INFO - (Background on this error at: https://sqlalche.me/e/14/2j85)
[2025-06-16T14:38:18.022+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.023+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.024+0000] {subprocess.py:93} INFO - üì• Reading /opt/airflow/data/raw/olist_order_items_dataset.csv
[2025-06-16T14:38:18.026+0000] {subprocess.py:93} INFO - üßπ Cleaning 'order_items' dataset...
[2025-06-16T14:38:18.027+0000] {subprocess.py:93} INFO - üóÑÔ∏è  Loading 'order_items' into PostgreSQL...
[2025-06-16T14:38:18.029+0000] {subprocess.py:93} INFO - ‚ùå ERROR while processing /opt/airflow/data/raw/olist_order_items_dataset.csv ‚Üí (psycopg2.errors.DependentObjectsStillExist) cannot drop table order_items because other objects depend on it
[2025-06-16T14:38:18.030+0000] {subprocess.py:93} INFO - DETAIL:  view stg_order_items depends on table order_items
[2025-06-16T14:38:18.031+0000] {subprocess.py:93} INFO - HINT:  Use DROP ... CASCADE to drop the dependent objects too.
[2025-06-16T14:38:18.032+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.034+0000] {subprocess.py:93} INFO - [SQL:
[2025-06-16T14:38:18.035+0000] {subprocess.py:93} INFO - DROP TABLE order_items]
[2025-06-16T14:38:18.036+0000] {subprocess.py:93} INFO - (Background on this error at: https://sqlalche.me/e/14/2j85)
[2025-06-16T14:38:18.037+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.038+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.039+0000] {subprocess.py:93} INFO - üì• Reading /opt/airflow/data/raw/olist_order_payments_dataset.csv
[2025-06-16T14:38:18.041+0000] {subprocess.py:93} INFO - üßπ Cleaning 'order_payments' dataset...
[2025-06-16T14:38:18.042+0000] {subprocess.py:93} INFO - üóÑÔ∏è  Loading 'order_payments' into PostgreSQL...
[2025-06-16T14:38:18.044+0000] {subprocess.py:93} INFO - ‚ùå ERROR while processing /opt/airflow/data/raw/olist_order_payments_dataset.csv ‚Üí (psycopg2.errors.DependentObjectsStillExist) cannot drop table order_payments because other objects depend on it
[2025-06-16T14:38:18.046+0000] {subprocess.py:93} INFO - DETAIL:  view stg_order_payments depends on table order_payments
[2025-06-16T14:38:18.047+0000] {subprocess.py:93} INFO - HINT:  Use DROP ... CASCADE to drop the dependent objects too.
[2025-06-16T14:38:18.048+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.050+0000] {subprocess.py:93} INFO - [SQL:
[2025-06-16T14:38:18.051+0000] {subprocess.py:93} INFO - DROP TABLE order_payments]
[2025-06-16T14:38:18.053+0000] {subprocess.py:93} INFO - (Background on this error at: https://sqlalche.me/e/14/2j85)
[2025-06-16T14:38:18.054+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.055+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.056+0000] {subprocess.py:93} INFO - üì• Reading /opt/airflow/data/raw/olist_order_reviews_dataset.csv
[2025-06-16T14:38:18.057+0000] {subprocess.py:93} INFO - üßπ Cleaning 'order_reviews' dataset...
[2025-06-16T14:38:18.059+0000] {subprocess.py:93} INFO - üóÑÔ∏è  Loading 'order_reviews' into PostgreSQL...
[2025-06-16T14:38:18.060+0000] {subprocess.py:93} INFO - ‚ùå ERROR while processing /opt/airflow/data/raw/olist_order_reviews_dataset.csv ‚Üí (psycopg2.errors.DependentObjectsStillExist) cannot drop table order_reviews because other objects depend on it
[2025-06-16T14:38:18.062+0000] {subprocess.py:93} INFO - DETAIL:  view stg_order_reviews depends on table order_reviews
[2025-06-16T14:38:18.063+0000] {subprocess.py:93} INFO - HINT:  Use DROP ... CASCADE to drop the dependent objects too.
[2025-06-16T14:38:18.064+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.066+0000] {subprocess.py:93} INFO - [SQL:
[2025-06-16T14:38:18.068+0000] {subprocess.py:93} INFO - DROP TABLE order_reviews]
[2025-06-16T14:38:18.069+0000] {subprocess.py:93} INFO - (Background on this error at: https://sqlalche.me/e/14/2j85)
[2025-06-16T14:38:18.070+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.071+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.072+0000] {subprocess.py:93} INFO - üì• Reading /opt/airflow/data/raw/olist_products_dataset.csv
[2025-06-16T14:38:18.074+0000] {subprocess.py:93} INFO - üßπ Cleaning 'products' dataset...
[2025-06-16T14:38:18.075+0000] {subprocess.py:93} INFO - üóÑÔ∏è  Loading 'products' into PostgreSQL...
[2025-06-16T14:38:18.076+0000] {subprocess.py:93} INFO - ‚ùå ERROR while processing /opt/airflow/data/raw/olist_products_dataset.csv ‚Üí (psycopg2.errors.DependentObjectsStillExist) cannot drop table products because other objects depend on it
[2025-06-16T14:38:18.077+0000] {subprocess.py:93} INFO - DETAIL:  view stg_products depends on table products
[2025-06-16T14:38:18.078+0000] {subprocess.py:93} INFO - HINT:  Use DROP ... CASCADE to drop the dependent objects too.
[2025-06-16T14:38:18.080+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.081+0000] {subprocess.py:93} INFO - [SQL:
[2025-06-16T14:38:18.082+0000] {subprocess.py:93} INFO - DROP TABLE products]
[2025-06-16T14:38:18.083+0000] {subprocess.py:93} INFO - (Background on this error at: https://sqlalche.me/e/14/2j85)
[2025-06-16T14:38:18.084+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.085+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.086+0000] {subprocess.py:93} INFO - üì• Reading /opt/airflow/data/raw/olist_sellers_dataset.csv
[2025-06-16T14:38:18.087+0000] {subprocess.py:93} INFO - üßπ Cleaning 'sellers' dataset...
[2025-06-16T14:38:18.088+0000] {subprocess.py:93} INFO - üóÑÔ∏è  Loading 'sellers' into PostgreSQL...
[2025-06-16T14:38:18.089+0000] {subprocess.py:93} INFO - ‚ùå ERROR while processing /opt/airflow/data/raw/olist_sellers_dataset.csv ‚Üí (psycopg2.errors.DependentObjectsStillExist) cannot drop table sellers because other objects depend on it
[2025-06-16T14:38:18.090+0000] {subprocess.py:93} INFO - DETAIL:  view stg_sellers depends on table sellers
[2025-06-16T14:38:18.091+0000] {subprocess.py:93} INFO - HINT:  Use DROP ... CASCADE to drop the dependent objects too.
[2025-06-16T14:38:18.092+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.093+0000] {subprocess.py:93} INFO - [SQL:
[2025-06-16T14:38:18.094+0000] {subprocess.py:93} INFO - DROP TABLE sellers]
[2025-06-16T14:38:18.095+0000] {subprocess.py:93} INFO - (Background on this error at: https://sqlalche.me/e/14/2j85)
[2025-06-16T14:38:18.096+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.097+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.099+0000] {subprocess.py:93} INFO - üì• Reading /opt/airflow/data/raw/product_category_name_translation.csv
[2025-06-16T14:38:18.101+0000] {subprocess.py:93} INFO - üßπ Cleaning 'category_translation' dataset...
[2025-06-16T14:38:18.103+0000] {subprocess.py:93} INFO - üóÑÔ∏è  Loading 'category_translation' into PostgreSQL...
[2025-06-16T14:38:18.104+0000] {subprocess.py:93} INFO - ‚ùå ERROR while processing /opt/airflow/data/raw/product_category_name_translation.csv ‚Üí (psycopg2.errors.DependentObjectsStillExist) cannot drop table category_translation because other objects depend on it
[2025-06-16T14:38:18.106+0000] {subprocess.py:93} INFO - DETAIL:  view stg_category_translation depends on table category_translation
[2025-06-16T14:38:18.107+0000] {subprocess.py:93} INFO - HINT:  Use DROP ... CASCADE to drop the dependent objects too.
[2025-06-16T14:38:18.108+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.110+0000] {subprocess.py:93} INFO - [SQL:
[2025-06-16T14:38:18.111+0000] {subprocess.py:93} INFO - DROP TABLE category_translation]
[2025-06-16T14:38:18.112+0000] {subprocess.py:93} INFO - (Background on this error at: https://sqlalche.me/e/14/2j85)
[2025-06-16T14:38:18.114+0000] {subprocess.py:93} INFO - 
[2025-06-16T14:38:18.354+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-06-16T14:38:18.366+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-16T14:38:18.578+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_brazil_pipeline, task_id=run_etl_script, run_id=manual__2025-06-16T14:37:26.407631+00:00, execution_date=20250616T143726, start_date=20250616T143742, end_date=20250616T143818
[2025-06-16T14:38:18.692+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-06-16T14:38:18.792+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-16T14:38:18.797+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
